R/addins.R

```diff
@@ -94,3 +96,78 @@ style_selection <- function() {
 get_rstudio_context <- function() {
   rstudioapi::getActiveDocumentContext()
 }
+
+
+# Dedicated binding for package styling addin. Simple wrapper calling style_pkg
+# with the selected addins style.
+style_package <- function() {
```

Comment:
I think we don't want another Addin, see also #250.

Reply:
Actually thanks go @jonmcalder we have figured out that you have not registered a new Addin, just moved existing code. I overlooked that. Sorry. Will add in #500 again.

Reply:
I thought this was intentional, as discussed when integrating your tweaks in https://github.com/miraisolutions/styler/pull/1#discussion_r256412827

Reply:
Well, the confusion arose because people wanted to add other Addins like #476 and #250 and in the diff, it looked as if you *added* an Addin for package styling. This was not the case, you just moved some code. So that was my mistake.

Reply:
I see, no problem!